### DISCUSSION TITLE
Presentation Mode toggle turn screen black

### DISCUSSION THREAD

**Discussion by matrdr** [sumatrapdfreader/sumatrapdf#2819] [General]
I tried out Sumatra PDF in presentation mode using a presenter remote from Logitech.
Nearly everything works fine but turning the screen black is only possible one way. So if you press the black screen button on the remote there is no way to undo that from the remote.

The remote would rely on the shortcut `.` for toggling the state of the black screen. But in Sumatra v3.4.6 the shortcut `.` does only activate black screen and deactivation must be done with `esc`.

Curiously in an older version of Sumatra toggling with `.` actually seemed to be possible. Was this feature purposefully removed? I liked it. And if yes, can I set an option somehow to reactivate it?

**Comment by GitHubRulesOK** [comment]
w - in presentation mode, make whole screen white
. - in presentation mode, make whole screen black

however have to agree there are odd behaviours since b and , were alternatives and testing between the 4 is now confusing
3.1.2 was primarily based on
PowerPoint users / presentation pointers behaviour
>Press B or Period to black out your screen / revive a currently blacked out screen. 
>Press W or Comma to white out your screen / bring it back.
so in 3.1.2 Keys B W and `.` were all toggles on/off but comma was not assigned ?

@kjk moving to issues 
there is
CmdPresentationWhiteBackground = w
CmdPresentationBlackBackground = .

and unlisted Toggle BlackBackground = b
however in pre release there is no "UnWhite" just using old shortcut key `,` now switches whiteout off
